packages/client/src/generation/TSClient/TSClient.ts

```diff
@@ -129,14 +129,12 @@ ${buildWarnEnvConflicts(edge, runtimeDir, runtimeName)}
 const PrismaClient = getPrismaClient(config)
 exports.PrismaClient = PrismaClient
 Object.assign(exports, Prisma)
+${this.options.runtimeName === 'edge' ? 'export { exports as default, Prisma, PrismaClient }' : ''}
 ${buildNFTAnnotations(dataProxy, engineType, platforms, relativeOutdir)}
 `
     return code
   }
   public toTS(edge = false): string {
-    // edge exports the same ts definitions as the index
-    if (edge === true) return `export * from './index'`
```

Comment:
We can work on not exposing node types and importing `util` which is a side-effect of bundling the `sql-template-tag` library. We could easily write a wrapper around it to make sure these type imports don't leak into the final bundle.

Reply:
OK. I'll try to make `runtime/index.d.ts` not requiring Node.js types

When I tried that last week, I was able to get rid of `util` dependency, but `api-extractor` sill added `/// <reference types="node" />` directive at the top of the bundled file (even when there's no node.js type reference in the output), and that part errored in Deno. Do you think it's ok to replace that `/// <reference ...>` part after processing it via `api-extractor`?

Reply:
If we change the type of `QueryEngineConfig['env']` (in `engine-core/src/common/types/QueryEngine.ts`) from `NodeJS.ProcessEnv | Record<string, string>` to `Record<string, string | undefined>`, then node.js type dependency disappears. Do you think it's an acceptable change?

Reply:
Sure, totally acceptable.